packages/elements-core/src/components/Docs/HttpService/SecuritySchemes.tsx

```diff
@@ -1,128 +1,64 @@
-import { Panel } from '@stoplight/mosaic';
-import { HttpSecurityScheme, IOauth2Flow, IOauth2SecurityScheme, IOauthFlowObjects } from '@stoplight/types';
-import { entries, keys, sortBy } from 'lodash';
+import { Box, Panel, PanelProps } from '@stoplight/mosaic';
+import { HttpSecurityScheme } from '@stoplight/types';
+import sortBy from 'lodash/sortBy';
 import React from 'react';
 
-import {
-  getReadableSecurityName,
-  isOauth2AuthorizationCodeFlow,
-  isOauth2ClientCredentialsOrPasswordFlow,
-  isOAuth2ImplicitFlow,
-  shouldIncludeKey,
-} from '../../../utils/oas/security';
+import { getDefaultDescription } from '../../..//utils/securitySchemes';
```

Comment:
double `/` here

Reply:
Fixed in a0bfbda32c1f49e73488839a7dbbb7e38f8c1780